Routers/BookingRouter.cs

```diff
@@ -0,0 +1,118 @@
+﻿using AirportTicketBooking.Classes;
+using AirportTicketBooking.Repositories;
+using ClassType = TicketClassType.ClassType;
+
+namespace AirportTicketBooking.Routers
+{
+    internal class BookingRouter
+    {
+        private BookingRepositoty _bookingRepositoty;
+        private FlightRouter _flightRouter;
+
+        public BookingRouter()
+        {
+            _bookingRepositoty = new BookingRepositoty();
+        }
+
+        public List<Booking> GetAllData()
+        {
+            return _bookingRepositoty.Load();
+        }
+
+        public List<Booking> GetAll()
+        {
+            return GetAllData().Where(booking => booking.ValidationMessage == null).ToList();
+        }
+
+        public bool TryGetByBookingId(int bookingId, out Booking booking)
+        {
+            booking = GetAll().FirstOrDefault(booking => booking.BookingId == bookingId);
```

Comment:
I think you should filter on the ValidationMessage here as what you did in the GetAll. The result of this method should be a part of the GetAll method.

Reply:
sorry, I didn't understand it, I used the data that returns from GetAll() so already the validationMassege is null

Reply:
instead of doing the filtration twice, we can do it once?